### PR TITLE
Switched copyright to NUnit Project

### PR DESCRIPTION
**Pull request by rprouse**
See nunit/governance#12

### CODE REVIEW

File: NOTICES.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 NUnit 3.0 is based on earlier versions of NUnit, with Portions
```
Comment (CharliePoole) [inline NOTICES.txt]: I didn't realize  that NUnit 3 used this NOTICES file that I created when transitioning from V2 to V3, in order to give credit to the past developers of NUnit V2. It doesn't really have anything to do with the old code, so I'd remove it.

**Review by CharliePoole — CHANGES_REQUESTED** [review]
Rather than changing the NOTICES file I think it should be removed.

**Comment by rprouse** [comment]
Notices removed

**Comment by CharliePoole** [comment]
Travis failures are unrelated

**Pull request merged**